moped-editor/src/views/projects/projectsListView/useProjectListViewQuery/useProjectListViewQuery.js

```diff
@@ -0,0 +1,54 @@
+import { useMemo } from "react";
+import { gql } from "apollo-boost";
+import { usePagination } from "./usePagination";
+import { useOrderBy } from "./useOrderBy";
+
+export const useGetProjectListView = ({ columnsToReturn }) => {
+  const { queryLimit, setQueryLimit, queryOffset, setQueryOffset } =
+    usePagination({ defaultLimit: 250, defaultOffset: 0 });
```

Comment:
Maybe these default values should be set outside this `useGetProjectListView` hook and passed in as params? Same with the order hook a few lines below this.

Reply:
they could be passed in from the PROJECT_LIST_VIEW_QUERY_CONFIG

Reply:
perfect! i forgot that `rowsPerPageOptions` is already in there too. I'll make this update - thanks!